includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -731,8 +731,7 @@ public function payment_fields() {
 		try {
 			$display_tokenization = $this->supports( 'tokenization' ) && ( is_checkout() || is_add_payment_method_page() );
 
-			wp_localize_script( 'WCPAY_CHECKOUT', 'wcpay_config', $this->get_payment_fields_js_config() );
-			wp_enqueue_script( 'WCPAY_CHECKOUT' );
+			add_action( 'wp_footer', [ $this, 'enqueue_payment_scripts' ] );
```

Comment:
Based on the comments on forum link (https://wordpress.org/support/topic/issue-with-wp_localize_script-or-similar-on-block-themes/), it looks like the recommended approach is to use `wp_add_inline_script` to enqueue script data (in preference to `wp_localize_script`).

@RadoslavGeorgiev should we be using `wp_add_inline_script` ?